source/extensions/common/dynamic_forward_proxy/dns_cache_impl.cc

```diff
@@ -353,8 +354,9 @@ void DnsCacheImpl::finishResolve(const std::string& host,
     }
     if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.use_dns_ttl")) {
       // Arbitrarily cap DNS re-resolution at 5s to avoid constant DNS queries.
```

Comment:
Looks like you may want to update this comment as well.

Reply:
Oops yup thanks will fix.